client/scripts/stores/Store.ts

```diff
@@ -9,7 +9,7 @@ abstract class Store<T> {
   public remove(item: T, eqFn?: (a: T) => boolean): Store<T> {
     const index = eqFn ? this._store.findIndex(eqFn) : this._store.indexOf(item);
     if (index === -1) {
-      console.error('Attempting to remove an object that was not found in the store');
+      console.error('Attempting to remove an object that was not found in the store:');
```

Comment:
why add the : if we are not printing the object?

Reply:
Yeah good catch, this was remnant of an old console.log. I'd deleted the log but not the colon.

Reply:
Yeah good catch, this was remnant of an old console.log. I'd deleted the log but not the colon.